src/app/model/Organization.ts

```diff
@@ -74,6 +74,16 @@ class Organization extends BaseModel {
       return { clientUid: "sb", email: "" };
     }
   }
+
+  logDonation(donation: DonationLog) {
```

Comment:
As a just in case I'm logging donations. We don't use them for anything but better to track in the event we need them